x/auction/keeper/auctions.go

```diff
@@ -0,0 +1,337 @@
+package keeper
+
+import (
+	"fmt"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/supply"
+	"github.com/kava-labs/kava/x/auction/types"
+)
+
+// StartForwardAuction starts a normal auction that mints the sold coins.
```

Comment:
worth noting that all bids are burned for Forward Auctions - different than how someone might expect the auction to work.

Reply:
added more comments onto the auction types